frontends/concrete-python/concrete/fhe/tracing/tracer.py

```diff
@@ -382,6 +382,12 @@ def _trace_numpy_operation(operation: Callable, *args, **kwargs) -> "Tracer":
             dtype = kwargs.get("dtype", np.int64)
             return Tracer(Node.constant(np.zeros(args[0].shape, dtype=dtype)), [])
 
+        if operation == np.expand_dims:
+            assert len(args) in {1, 2}
+            if len(args) == 2:
+                kwargs["axis"] = args[1]
```

Comment:
Maybe using:
https://docs.python.org/3/library/inspect.html#introspecting-callables-with-the-signature-object
https://docs.python.org/3/library/inspect.html#inspect.Signature.bind
provide a more generic way to bind the parameters and to verify call correctness (vs the assert)

Reply:
Not necessary for now, we have very small number of exceptional cases to our logic.